### DISCUSSION TITLE
Windows 7 - Not supported?

### DISCUSSION THREAD

**Discussion by expyth** [FreeTubeApp/FreeTube#5382] [Q&A]
Freetube 0.17 beta stopped loading videos on all invidious set as default  (sometimes this would work in the past). I downloaded the latest verison, setup file won't launch anything. I downloaded as freetube-0.21.0-win-x64-portable.zip instead, it gave an error that the application us not a win32. My OS is 64-bit.

I was trying to find the system requirements, so according the home page it works only on Win10 and later.

So, I am out of luck? Any tips?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Check out this thread to get FreeTube working on Win7: https://github.com/FreeTubeApp/FreeTube/discussions/4848#discussioncomment-9012714

**Reply by expyth** [reply]
Life saver, thank you!!!
